model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -288,6 +288,35 @@ class PackageCurationTest : WordSpec({
 
             curatedPkg.pkg.isModified shouldBe false
         }
+
+        "work with Ivy version ranges" {
+            val pkg = Package(
```

Comment:
Would `Package.EMPTY.copy()` be ok in this case?

Reply:
Done.